cassandra/cqlengine/connection.py

```diff
@@ -222,7 +228,14 @@ def set_session(s):
     This may be relaxed in the future
     """
 
-    conn = get_connection()
+    try:
+        conn = get_connection()
+    except CQLEngineException:
+        # no default connection set; initalize one
+        default_name = 'default'
+        conn = Connection.from_session(default_name, s)
+        _connections[default_name] = conn
+        set_default_connection[default_name]
```

Comment:
This doesn't seem to work as expected. I'm getting this exception:
```
    connection.set_session(s)
  File "/home/aboudreault/dev/python-driver/cassandra/cqlengine/connection.py", line 238, in set_session
    set_default_connection[default_name]
TypeError: 'function' object has no attribute '__getitem__'
```
You probably want to do set_default_connection(...).